assets/ts/schedule/components/line-diagram/OtherStopList.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { ReactElement } from "react";
+import { RouteStop, StopTree } from "../__schedule";
+import StopCard from "./StopCard";
+import { Alert } from "../../../__v3api";
+
+interface Props {
+  alerts: Alert[];
+  handleStopClick: (stop: RouteStop) => void;
+  otherRouteStops: RouteStop[];
+  searchQuery?: string;
+  stopTree: StopTree | null;
+}
+
+const stopCountWords = (stopCount: number): string =>
+  stopCount === 1 ? `1 Stop` : `${stopCount} Stops`;
+
+const OtherStopList = ({
+  alerts,
+  handleStopClick,
+  otherRouteStops,
+  searchQuery,
+  stopTree
+}: Props): ReactElement<HTMLElement> =>
+  otherRouteStops.length > 0 ? (
+    <>
```

Comment:
suggestion: you don't need an empty wrapper here as it's already wrapping a single element

Reply:
[PTALA](https://github.com/mbta/dotcom/pull/2454/commits/1d3612ebf970049c51dbe630592a33ec73ee65f2?diff=split&w=1)